apps/frontend/src/pages/quiz/styles.module.css

```diff
@@ -1,10 +1,14 @@
 .container {
+	position: relative;
 	height: 100dvh;
 	padding: 15px;
+	overflow: hidden;
 	background-color: var(--background);
 }
 
 .introduction {
+	position: relative;
+	z-index: 1;
```

Comment:
Move z-index to variables.css. Create variables like `--z-index-low`, `--z-index-medium`, `--z-index-high`